### DISCUSSION TITLE
Gostaria de entender como é a conexão com banco de dados

### DISCUSSION THREAD

**Discussion by oimarcella** [balta-io/forum#549] [Tira Dúvidas] [answered]
Eu tentei executar um SELECT na minha tabela de alunos e dava erro pois não encontrava. Até eu perceber que no Azure Data Studio eu precisava alterar o database para balta, pois estava na master. E a db balta foi oq foi criado desde o início do curso de banco de dados.

1- Mas o que é então esse database master? 
2 - O server, por que localhost? Ele é referente ao docker com container rodando na minha máquina?
3 - O que é o server group? 

Isso ficou um pouco confuso para mim, tanto que até para fazer a conexão preciso olhar uma cola, porque nao entendo oq colocar em cada campo pois não fez sentido para mim.

![image](https://github.com/balta-io/forum/assets/43733159/aea7901e-6828-49f3-aca3-a44b781519ee)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @mlasa, como vai?

1- Mas o que é então esse database master?
- O database master é uma base padrão do SQL Server que também é usada para manter configurações gerais do servidor. Como todo servidor tem esta base, se não informamos um banco, ele sempre se conecta nessa por padrão.
- É importante ressaltar que não devemos criar tabelas aí.

2 - O server, por que localhost? Ele é referente ao docker com container rodando na minha máquina?
- Localhost significa que está rodando local, na sua máquina. Se fosse se conectar a um servidor na internet, seria um endereço de IP (84.36.12.20) ou um DNS (balta.io, azure.com, etc...)

3 - O que é o server group?
- É uma configuração local para agrupar servidores... isto fica apenas na sua máquina. Vamos supor que você tenha 10 bancos SQL cadastrados e queira separar eles por LOCAL... você pode criar dois ou mais grupos informando estes locais... como Local ou Localhost, Produção, Internet, etc...

Qualquer outra dúvida é só chamar!
